scipy/constants/_codata.py

```diff
@@ -465,6 +476,23 @@
 Wien frequency displacement law constant               5.878 933 e10         0.000 010 e10         Hz K^-1
 Wien wavelength displacement law constant              2.897 7685 e-3        0.000 0051 e-3        m K"""
 
+
+def exact2006(exact):
+    mu0 = 4e-7 * math.pi
+    c = exact['speed of light in vacuum']
+    epsilon0 = 1 / (mu0 * c**2)
+    replace = {
+        'mag. constant': mu0,
+        'electric constant': epsilon0,
+        'atomic unit of permittivity': 4*math.pi*epsilon0,
```

Comment:
Here's an example of a value that doesn't seem to be listed where I'd expect it. For instance, it's not even listed in the PDF [Extensive Listing](https://physics.nist.gov/cuu/pdf/all_2006.pdf).

Reply:
> Here's an example of a value for which I couldn't easily find the official formula in CODATA references. For instance, the quantity is not even listed in the 2006 PDF [Extensive Listing](https://physics.nist.gov/cuu/pdf/all_2006.pdf). Where did these formulas for computing derived values come from?

- The `mag. constant`, `electric constant`, `characteristic impedance of vacuum` can be found in the extensive listings.
- The `atomic unit of permittivity` can´t be found in the extensive listing of the CODATA values but in the accompanying scientific literature even though a different formula is given. https://en.wikipedia.org/wiki/Atomic_units quotes the formula used here.
- The relationships between different units is given as well in the accompanying documents. https://physics.nist.gov/cuu/pdf/factors_2018.pdf
- So, I think these formulas are good to go.

> Sorry about all the noise in this PR @pv; I imagine you're getting a lot of notifications. But do you remember where you got these formulas from?
> 
> @jakobjakobson13 Do you know where these are defined? Would you be willing to take a look at the current state of things?
>
> I think this is the only thing stopping this from merging, (although the assertions protect against them from being blatantly wrong, so it still seems quite safe). Everything else LGTM.

Reply:
> The atomic unit of permittivity can´t be found in the extensive listing of the CODATA values but in the accompanying scientific literature...

I understand your response (that I can find formulas elsewhere online), but I don't understand why referring to another reference would be OK. IIUC, CODATA is supposed to be an _authoritative_ source of these values, right? So if I recalculate the value according to some formula I found *anywhere* on the internet, how can I be sure that this is the formula CODATA authors had in mind when it declared that the value to be exact? Isn't there an authoritative source of the formulas of the constants that are defined exactly in terms of others?